main.go

```diff
@@ -5,14 +5,16 @@ package main
 import (
 	"flag"
 	"fmt"
+	"os"
+
 	"github.com/genomagic/master"
 	"github.com/genomagic/prepper"
 )
 
 func main() {
 	fileParam := "fastq"
 	fileValues := "raw_sequence.fastq"
-	fileUsage := "the path to the FASTQ file containing raw sequence data for assembly"
+	fileUsage := "*REQUIRED* the path to the FASTQ file containing raw sequence data for assembly"
```

Comment:
I would not add something like this in the usage of the param. Check the documentation of the `flag` package and see whether it provides something like this.

Reply:
I check flag package and unfortunately it doesn't provide ability for certain flags to be required.

Reply:
Alright, then I suggest removing the `*REQUIRED*` since the `panic` already mentions that `-fastq` is required.

Reply:
Fixed. eef0cf0